airtime_mvc/application/controllers/ApiController.php

```diff
@@ -523,6 +523,7 @@ public function stationMetadataAction()
             $result["description"] = Application_Model_Preference::GetStationDescription();
             $result["timezone"] = Application_Model_Preference::GetDefaultTimezone();
             $result["locale"] = Application_Model_Preference::GetDefaultLocale();
+            $result["enabled_stream_urls"] = Application_Model_StreamSetting::getEnabledStreamUrls();
```

Comment:
Calling it "enabled_stream_urls" in the public API seems verbose. Would just calling it "stream_urls" still make sense?